eos/form-factors/pi-lcdas.cc

```diff
@@ -56,9 +56,9 @@ namespace eos
             a2pi_0(p["pi::a2@1GeV"], u),
             a4pi_0(p["pi::a4@1GeV"], u),
             f3pi_0(p["pi::f3@1GeV"], u),
-            omega3pi_0(p["pi::omega3@1GeV"], u),
-            deltapipi_0(p["pi::delta^2@1GeV"], u),
-            omega4pi_0(p["pi::omega4@1GeV"], u),
+            omega3_0(p["pi::omega3@1GeV"], u),
+            delta4_0(p["pi::delta^2@1GeV"], u),
```

Comment:
This parameter name is misleading ^^

Reply:
In what way? The numbers indicate the twist of the corresponding operator. I( have no idea who first came up with ``deltapipi`` in the first place, but I can track it back to my original implementation mirroring Senti's Mathematica code.)

Reply:
The `2` in the parameter name looks error prone wrt to the `4` in the ParameterUser instance.

Reply:
@cbolognani is going to introduce a new commit at the top of the branch that renames ``pi::delta^2`` -> ``pi::delta4`` and ``K::delta^2`` -> ``pi::delta4`` everywhere, i.e.:
  - ``eos/form-factors/analytic-b-to-psd-dkmmo2008-impl.hh``
  - ``eos/form-factors/pi-lcdas.cc``
  - ``eos/form-factors/kaon-lcdas.cc``
  - ``eos/parameters/hadron-properties.yaml``
 
and any test cases that might be affected

Reply:
This commit has been added. Resolved.